.github/workflows/scripts/check_for_redirects.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  /**
+   * Add redirects-needed label if count of deleted files is greater than 0
+   *
+   * @param {Object} obj.artifactName - Name of artifiact file to check
+   */
+  addRedirectsNeededLabel: async ({ github, context, fs, artifactName }) => {
+    const {
+      payload: {
+        repository: {
+          owner: { login: ownerLogin },
+          name: repoName
+        }
+      }
+    } = context;
+
+    const artifactContents = fs
+      .readFileSync(`./${artifactName}.txt`)
+      .toString();
+
+    const [prNumberStr, numOfDeletedFilesStr] = artifactContents.split('\n');
```

Comment:
Just for abundance of caution since this runs on `workflow_run`, can we put some sanitization here? e.g.

https://github.com/aws-amplify/amplify-ui/blob/d1a316fd4103cceb6f01622b9ec3c82c6de121d1/.github/workflows/test-fork-prs.yml#L108-L109

Reply:
I think it's a good general practice to to sanitize any inputs we get via artifacts.

Reply:
For my use case, do you think it would be okay to just check if the two values I'm using are actually numbers with `Number()` and `isNaN`?

For instance:
```js
const prNumber = Number(prNumberStr);
const numOfDeletedFiles = Number(numOfDeletedFilesStr);

if (!isNaN(prNumber) && !isNaN(numOfDeletedFiles)) {
  console.log('PR number that triggered workflow:', prNumberStr);
  console.log('Number of deleted files:', numOfDeletedFilesStr);

  if (numOfDeletedFiles > 0) {
     github.rest.issues.addLabels({
       owner: ownerLogin,
       repo: repoName,
       issue_number: prNumber,
       labels: ['redirects-needed']
     });
   }
}
```

Reply:
Talked offline to use explicit regex test, resolving.